<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Hemanth" failures="1" tests="1" name="Test" time="9.958" errors="0" timestamp="2022-08-19T13:08:57 IST">
  <testcase classname="com.AdminModule2TestCasess.RegisterAppartmentAndVerifyInDetailsPage" name="appartment" time="1.629">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [3] but found [2]
at com.AdminModule2TestCasess.RegisterAppartmentAndVerifyInDetailsPage.appartment(RegisterAppartmentAndVerifyInDetailsPage.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- appartment -->
</testsuite> <!-- Test -->
